src/pinns_pde_solve.jl

```diff
@@ -20,20 +21,20 @@ struct PhysicsInformedNN{C,P,PH,DER,T,K}
 end
 
 function PhysicsInformedNN(chain,
-                           init_params = nothing;
+                           strategy;
+                           _initθ = nothing,
```

Comment:
Keep the old name. We shouldn't require Unicode in the API otherwise some terminals won't be able to use the argument.

Reply:
Actually, it already was `initθ` in arguments of PhysicsInformedNN. Ok, I will rename all the `initθ` to `init_params`.

Reply:
Yeah internally is fine, but we have a policy to not expose Unicode externally.

Reply:
ok